avm/res/service-bus/namespace/topic/main.bicep

```
@@ -142,6 +146,29 @@ resource topic_roleAssignments 'Microsoft.Authorization/roleAssignments@2022-04-
   scope: topic
 }]
 
+module topic_subscription 'subscription/main.bicep' = [for (subscription, index) in (subscriptions ?? []): {
+  name: '${deployment().name}-subscription-${index}'
+  params: {
+    name: subscription.name
+    namespaceName: namespace.name
+    topicName: topic.name
+    autoDeleteOnIdle: subscription.?autoDeleteOnIdle ?? 'PT1H'
+    defaultMessageTimeToLive: subscription.?defaultMessageTimeToLive ?? 'P14D'
+    duplicateDetectionHistoryTimeWindow: subscription.?duplicateDetectionHistoryTimeWindow ?? 'PT10M'
+    enableBatchedOperations: subscription.?enableBatchedOperations ?? true
+    clientAffineProperties: subscription.?clientAffineProperties ?? {}
+    deadLetteringOnFilterEvaluationExceptions: subscription.?deadLetteringOnFilterEvaluationExceptions ?? true
+    deadLetteringOnMessageExpiration: subscription.?deadLetteringOnMessageExpiration ?? false
+    forwardDeadLetteredMessagesTo: subscription.?forwardDeadLetteredMessagesTo ?? null
+    forwardTo: subscription.?forwardTo ?? null
```

```suggestion
    forwardDeadLetteredMessagesTo: subscription.?forwardDeadLetteredMessagesTo
    forwardTo: subscription.?forwardTo
```

Not necessary as this would anyways result in `null`